docs/semantic-conventions.md

```diff
@@ -1,97 +1,99 @@
 # Semantic conventions
 
-This document describes which [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions)
+This document describes
+which [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions)
 are implemented by Java autoinstrumentation and which ones are not.
 
 ## Http Server
 
-| Attribute | Required | Implemented? |
-|---|:---:|:---:|
-| `http.method` | Y | + |
-| `http.url` | N | - [1] |
-| `http.target` | N | + [1] |
-| `http.host` | N | + [1] |
-| `http.scheme` | N | + [1] |
-| `http.status_code` | Y | + |
-| `http.flavor` | N | + [2] |
-| `http.user_agent` | N | + |
-| `http.request_content_length` | N | - |
-| `http.request_content_length_uncompressed` | N | - |
-| `http.response_content_length` | N | - |
-| `http.response_content_length_uncompressed` | N | - |
-| `http.server_name` | N | - |
-| `http.route` | N | - |
-| `http.client_ip` | N | + |
+| Attribute                                   | Required | Implemented? |
+|---------------------------------------------|:--------:|:------------:|
+| `http.method`                               |    Y     |      +       |
+| `http.url`                                  |    N     |    - [1]     |
+| `http.target`                               |    N     |    + [1]     |
+| `http.host`                                 |    N     |    + [1]     |
+| `http.scheme`                               |    N     |    + [1]     |
+| `http.status_code`                          |    Y     |      +       |
+| `http.flavor`                               |    N     |    + [2]     |
+| `http.user_agent`                           |    N     |      +       |
+| `http.request_content_length`               |    N     |      -       |
+| `http.request_content_length_uncompressed`  |    N     |      -       |
+| `http.response_content_length`              |    N     |      -       |
+| `http.response_content_length_uncompressed` |    N     |      -       |
```

Comment:
```suggestion
| `http.response_content_length`              |    N     |      -       |
```

I think this document might need a separate PR to update it; I can work on it after we merge this one.